### PR TITLE
Ribotish quality

### PR DESCRIPTION
**Pull request by pinin4fjords**
Add test data for riboseq. Specifically for ribotish/quality in this instance.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -287,8 +287,12 @@ The folder is structured in the following way: Any nonspecific-pangenome file is
     - index
       - salmon: salmon index created with `transcriptome.fasta`
     - vep.tar.gz: Compressed VEP cache containing info.txt and synonyms of chr22 only. No annotations included.
+    - riboseq_expression
```
Comment (maxulysse) [inline README.md:290]: do we need a specific folder for riboseq, or can these files go where the other gtf and bam files are?

Comment (pinin4fjords) [reply README.md:290]: I wouldn't want people to use them like regular RNAseq or genomic alignments

Comment (maxulysse) [reply README.md:290]: sounds like a good enough reason for me

**Review by maxulysse — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (pinin4fjords) [reply README.md:290]: Thanks!

**Pull request merged**